### ISSUE TITLE
Receiver address needs to be verified on Production AWS SES server.

### ISSUE THREAD

**Issue by DevTheOther** [django-ses/django-ses#297]
I am using django-ses with AWS SES. I was expecting that once I transitioned out of the sandbox and into a production sever, it would no  longer require me to verify the email receiver. In order to send the email I need to verify the email using "manage.py ses_email_address --add [email_address]". Then the user receives the verify email, which must be verified. Only then can they receive emails. The sender email is already verified and is not a problem. 
Is this a limitation of the django-ses package?

**Comment by pcraciunoiu** [comment]
I don't think it's the package, it must be a setting or limitation in how AWS sends email on your account.

**Comment by DevTheOther** [comment]
But it doesn't even seem to be using the Amazon SES configurations, such as the verified email address. It has to be verified locally using manage.py ses_email_address --add [email_address]. 
I tried to remove django_ses from the installed apps, thinking that it is only using that if that app is installed, but it still seems to be using those configurations. Where is the email address list stored? I assume it is local.

**Comment by DevTheOther** [comment]
I figured it out. 

After digging deeper into the django-ses manager code, I saw that the verify_email was going to the AWS backend, but it took a while to realize that, even though I had the right path for the AWS_SES_REGION_ENDPOINT, the REGION of the AWS SES was us-east-2, not the default us-east-1. So apparently there is an us-east-1 instance that exists and is still in the sandbox. Once I defined the AWS_SES_REGION_NAME to us-east-2, it worked as expected. 

Thanks for the response.
